spec/system/forms/developer_search_form_spec.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Developer search', type: :system do
+  before do
+    @developer = create(:developer, email: 'phanthanhhai07t1@gmail.com')
+    @language = create(:language, code: 'en')
+    create(:language, code: 'vi')
+    @programming_language = create(:programming_language, name: 'ruby')
+    create(:programming_language, name: 'php')
+    @developer_language = create(:developer_language, developer_id: @developer.id, language_id: @language.id)
+    @developer_programming_language = create(:developer_programming_language, developer_id: @developer.id,
+                                                                              programming_language_id: @programming_language.id)
+  end
```

Comment:
Please insert a blank line between `end` and `it`.
Without a blank line, it looks very dense.

Reply:
@haiphantony 
I don't know what you mean by `it wrong with rubocop`, but can you share the message?

Reply:
It mean  as follows 
https://gyazo.com/4ca2fbc6a2efc72e282d6037b7e446af

Reply:
@haiphantony 
Sorry I overlooked your comment.
The error is this one, right?
http://www.rubydoc.info/github/bbatsov/RuboCop/RuboCop/Cop/Layout/EmptyLinesAroundBeginBody

What I asked you was `between end and it`.
So it should be as follows.
```
it ... do
   ...
end

it ... do
  ...
end
```
Not as follows.
```
it ... do

  ...

end
it ... do

  ...

end
```
I think it's common to add an blank line between methods, for example, for readability.

Reply:
Thanks, I understand